### DISCUSSION TITLE
Required for build orders widget

### DISCUSSION THREAD

**Discussion by gunstr** [inventree/InvenTree#8606] [Q&A] [answered]
I have used the "Low Stock" and "Required for build orders" features a lot to get an overview of what parts that needs to be ordered.

In PUI there is a "Low stock" widget that can be added to the dashboard but I cannot find any "Required for build orders" wiget. Is it still to be developed or is there any other way to list all parts to be ordered that I have not found?

**Comment by matmair** [comment]
@gunstr I do not think that was implemented

**Comment by SchrodingersGat** [comment, marked as answer]
I have created an issue against that here: https://github.com/inventree/InvenTree/issues/8609
